src/store/modules/loadingStatus.ts

```diff
@@ -0,0 +1,22 @@
+import { Action, Module, Mutation, VuexModule } from 'vuex-module-decorators'
+
+@Module({ namespaced: true })
+export default class loadingStatusModule extends VuexModule {
+  isLoading: boolean = false
```

Comment:
making two state will be good. one for global loading and another for component level (can use if needed). and global one only can use in app.vue . we can re-use same loading component for local state and global

Reply:
sounds good :)